app-vim/jedi/jedi-0.10.0.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7,3_8} )
+
+inherit vim-plugin python-any-r1
+
+DESCRIPTION="vim plugin: binding to the autocompletion library jedi"
+HOMEPAGE="https://github.com/davidhalter/jedi-vim"
+SRC_URI="https://github.com/davidhalter/jedi-vim/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	dev-python/jedi[${PYTHON_USEDEP}]
```

Comment:
Using *python-any-r1* eliminates the need for `PYTHON_USEDEP`. Instead you want to use 
`$(python_gen_cond_dep 'dev-python/jedi[${PYTHON_MULTI_USEDEP}]')`

Reply:
If I use that I get:

```
Appending /home/chymera/src/gentoo to PORTDIR_OVERLAY...
 * ERROR: app-vim/jedi-0.10.0::gentoo failed (depend phase):
 *   External commands disallowed while sourcing ebuild: python_gen_cond_dep dev-python/jedi[${PYTHON_MULTI_USEDEP}]
 *
 * Call stack:
 *            ebuild.sh, line 609:  Called source '/home/chymera/src/gentoo/app-vim/jedi/jedi-0.10.0.ebuild'
 *   jedi-0.10.0.ebuild, line  21:  Called command_not_found_handle 'python_gen_cond_dep' 'dev-python/jedi[${PYTHON_MULTI_USEDEP}]'
 *            ebuild.sh, line  89:  Called die
 * The specific snippet of code:
 *   		die "External commands disallowed while sourcing ebuild: ${*}"
 *
 * If you need support, post the output of `emerge --info '=app-vim/jedi-0.10.0::gentoo'`,
 * the complete build log and the output of `emerge -pqv '=app-vim/jedi-0.10.0::gentoo'`.
 * Working directory: '/usr/lib64/python3.6/site-packages'
 * S: '/var/tmp/portage/app-vim/jedi-0.10.0/work/jedi-0.10.0'
```

Reply:
Right, it's `python_gen_any_dep` with this eclass.